sn_node/src/bin/safenode/main.rs

```diff
@@ -360,6 +364,41 @@ You can check your reward balance by running:
     }
 }
 
+fn check_node_disk_space(root_dir: &Path) -> Result<()> {
+    let disks = Disks::new_with_refreshed_list();
+    let disks = disks.list();
+    let required_space_bytes = (MAX_RECORDS_COUNT * *MAX_CHUNK_SIZE) as u64;
```

Comment:
as mentioned at https://github.com/maidsafe/safe_network/commit/85c81e6c26c6ffcd50bcc5a82470ec87f9cafd21
the node_size shall be calculated as : `MAX_RECORDS_COUNT * (MAX_CHUNK_SIZE / 2)`

This calculation also appears duplicated in other place, hence better make it a helper facility